packages/overdrive/lib/components/StandardModal/StandardModal.tsx

```diff
@@ -60,6 +62,10 @@ export const StandardModal: FunctionComponent<Props> = ({
 		<Modal isOpen={isOpen} onRequestClose={onRequestClose}>
```

Comment:
Thanks for doing the "x" button thing. Just think we need to now optically align it—so the lhs padding _looks_ the same as the right.

![image](https://user-images.githubusercontent.com/599459/88354133-45a49a00-cda3-11ea-9f5f-9969c7ed95fc.png)

Think this can be achieved by simply removing the rhs padding?

Reply:
Done